helm-chart/kubray-operator/README.md

```diff
@@ -18,23 +18,23 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 Please use command below:
 ```console
-$ helm install  kubray-operator . --values values.yaml --namespace default --create-namespace
+$ helm install kubray-operator . --values values.yaml --namespace kubray-operator --create-namespace
```

Comment:
typo.  `kubray` -> `kuberay`. 

In other examples, we proposed to use namespace `ray-system`.   Besides the operator, we may have other components like kuberay apiserver or node detector later. I think it's better to use a different name than `kuberay-operator`.  what do you think?

Reply:
> typo. `kubray` -> `kuberay`.

Fixed.

> But why the dir is named `kubray-operator`?

---

> In other examples, we proposed to use namespace ray-system. Besides the operator, we may have other components like kuberay apiserver or node detector later. I think it's better to use a different name than kuberay-operator. what do you think?

I used `ray-system` in the other (ray cluster) examples.

---

BTW, it's better to remove `kube` or `kub` prefix IMO.

Prefixed with `kube` is sometimes the K8s system's privileges.